docker-hub/webhooks.md

```diff
@@ -9,6 +9,8 @@ cause an action in another application in response to an event in the
 repository. Docker Hub webhooks fire when an image is built in, or a new tag
 added to, your automated build repository.
 
+Configure webhooks on https://hub.docker.com/r/betalo/{repository}/~/settings/webhooks/ where '{repository}' is replaced with your Docker repository name.
```

Comment:
For placeholder text, please use `<REPOSITORY>`. You also need to have placeholder text for the username, so that other people don't try to use your username. Thanks!